charts/activemq/Chart.yaml

```diff
@@ -7,8 +7,8 @@ keywords:
 name: activemq
 sources:
   - https://github.com/Alfresco/alfresco-helm-charts
-version: 3.5.4
-appVersion: 5.18.4
+version: 3.5.5
```

Comment:
it's better if you release first activemq alone so other charts could pick this version